### ISSUE TITLE
[ACCESSIBILITY] - Each button needs unique labels for assistive technology users

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#503]
Describe the accessibility issue
Buttons needs clear and unambiguous labels and instructions. 

To Reproduce

Launch App
(ensure user is logged in/NetID connected)
Go to 'Wallet'
 
Expected behavior
When a screen reader user has this screen read by their assistive technology they should understand what each button is for. 

Actual behavior
When a screen reader user has this screen read by their assistive technology they hear "View button"  and then few seconds later "View button" leaving them unsure which "view button" is to what data Cash or Meal plan.

Screenshots
![Screenshot_20210615-112431_Illinois](https://user-images.githubusercontent.com/82892873/125552667-8958db9a-7b71-4b34-9d9a-240f76618b15.jpg)

Device: iPhone XR
OS: iOS 14.4

**Comment by acovher2** [comment]
Hello, 
It has been 2 months; are there any updates on this issue? 
Please provide any information you can. Thank you.

**Comment by acovher2** [comment]
Hello, 
It has been 2 months; are there any updates on this issue? 
Please provide any information you can. Thank you.

**Comment by TodorBachvarov** [comment]
For now the users that use the Accessibility feature get almost the same information as the other users. For each button we pronounce that it's a button and the title of the button. If a button leads the user to an external webpage outside of the app we place an icon and description "External link". If we want to add some specific hint to a button in order to improve the accessibility user experience I'll need a list of the exact text that should be pronounced and a button that should be applied to.

**Comment by acovher2** [comment]
We can not leave as is with no resolution to the issue. 

Please add detail to describe the button target. There are only 2 buttons I think the request for a list is a bit much. 
"**View Illini cash**" for first "View" and **"View Meal plan**" for second "View" will suffice I don't think a list is needed.
 
If you know of anywhere else in the app same label is used for buttons please do the same as the above and describe target accurately in what will be pronounced by screen reader. 

This was retested with 3.2.34 and still found to be present. Please address with the above resolution. 

Thank you,
Amanda

**Comment by TodorBachvarov** [comment]
Added hint to these two buttons. Added in version 3.3.9.

**Comment by acovher2** [comment]
will retest in that apk. Thanks. Please slack me if you have that apk ready. As of now, I've only seen 3.3..8 out. 
Thank you,
Amanda

**Comment by acovher2** [comment]
retested and resolved version 4.0.29
